src/App.js

```diff
@@ -1,18 +1,32 @@
 import React, { Fragment } from "react";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
-import { ChakraProvider } from "@chakra-ui/react";
+import { ChakraProvider, extendTheme } from "@chakra-ui/react";
 import Home from "./components/Home/Home";
+import About from "./components/Pages/About";
 import Contact from "./components/Pages/Contact";
 import "slick-carousel/slick/slick.css";
 import "slick-carousel/slick/slick-theme.css";
 
 function App() {
+  const colors = {
+    brand: {
+      900: "#1a365d",
+      800: "#153e75",
+      700: "#2a69ac",
+    },
+  };
+  const fonts = {
+    heading: "Open Sans",
+    body: "Open Sans",
+  };
```

Comment:
Nice idea. I like this one. 

Consider creating a file named "theme.js" that exports an object that contains the theme. 

That way it would be easy to see  how values we are using in the app